### PR TITLE
feat: add version info to footer (#4260)

### LINKED ISSUE #4260: [AnVIL Catalog] Add version info to footer
#### Depends On
[Add optional version information to footer](https://github.com/DataBiosphere/findable-ui/issues/258) and [Upgrade findable-ui to 15.0.0](https://github.com/anvilproject/anvil-portal/issues/3364)

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket

Closes #4260.

### Reviewers

@NoopDog.

### Changes

- Added version info into AnVIL catalog footer.

image

### CODE REVIEW

File: site-config/anvil-catalog/dev/config.ts [hunk 1]
```diff
@@ -16,15 +17,25 @@ import { buildNavigation } from "./layout/header/navigation/navigation";
...
 const PORTAL_URL = "https://anvilproject.dev.clevercanary.com";
 const SLOGAN = "NHGRI Analysis Visualization and Informatics Lab-space";
+const VERSION = process.env.NEXT_PUBLIC_VERSION;
 
 export function makeConfig(
   browserUrl: string,
   portalUrl: string,
-  explorerUrl: string
+  explorerUrl: string,
+  gitHubUrl: string = GIT_HUB_REPO_URL,
```
Comment (NoopDog) [inline site-config/anvil-catalog/dev/config.ts:33]: Hi @frano-m, can you do this without modifying the makeConfig? The provider or component can directly access the environment.

**Review by NoopDog — CHANGES_REQUESTED** [review]

File: site-config/anvil-catalog/dev/config.ts (hunk 1 above)
Comment (frano-m) [reply site-config/anvil-catalog/dev/config.ts:33]: Sure.

Comment (NoopDog) [reply site-config/anvil-catalog/dev/config.ts:33]: Oh I misread this... it's the repo URL.

**Review by NoopDog — APPROVED** [review]
Thanks @frano-m !

**Pull request merged**